bcgov/House-Innovations-HOUSNAV
# HOUSNAV-211: Building Type Analysis Page & Walkthrough

[HOUSNAV-211](https://hous-bssb.atlassian.net/browse/HOUSNAV-211)

## Overview & Purpose
Use the SME [document](https://docs.google.com/spreadsheets/d/1v5emSJu2xSsimbE28YvvM3AVd74Vfsjj/edit?gid=468521882#gid=468521882) to create the Building Type Analysis walkthrough

## Summary of Changes
- Updated the building type analysis json with questions and results
- Created a page at the building type analysis route
- Added features to the result display to show and hide different items to match the design
- Created a primary link variant

## Testing
- Click on Building Type Analysis on the homepage and see the new walkthrough
- Verify that the walkthrough logic matches the adjusted logic in the SME docuement
- Verify the results page displays correctly for each case

## Checklist
